test/UID2.Client.Test/EncryptionTestsV3.cs

```diff
@@ -32,6 +32,154 @@ public void SmokeTest()
             Assert.Equal(EXAMPLE_UID, res.Uid);
         }
 
+        //////////////////////  Sharing tests
+        [Fact]
+        public void CanEncryptAndDecryptForSharing()
+        {
+            var client = new UID2Client("endpoint", "authkey", CLIENT_SECRET, IdentityScope.UID2);
+            var json = KeySetToJsonForSharing(MASTER_KEY, SITE_KEY);
+            var refreshResult = client.RefreshJson(json);
+            Assert.True(refreshResult.Success);
+
+            var encrypted = client.Encrypt(EXAMPLE_UID);
+            Assert.Equal(EncryptionStatus.Success, encrypted.Status);
+            string advertisingToken = encrypted.EncryptedData;
+
+            var res = client.Decrypt(advertisingToken, NOW);
+            Assert.True(res.Success);
+            Assert.Equal(EXAMPLE_UID, res.Uid);
+        }
+        /*
+        [Fact]
+        public void EncryptKeyExpired()
```

Comment:
just confirming we wanna keep these commented after merge?

Reply:
oops, fixed